supervised/preprocessing/eda.py

```diff
@@ -162,3 +162,161 @@ def compute(X, y, eda_path):
             fout.close()
         except Exception as e:
             logger.error(f"There was an issue when running EDA. {str(e)}")
+
+    @staticmethod
+    def extensive_eda(df, y, save_path):
+
+        X = df.copy(deep=True)
```

Comment:
what if the dataset will be huge? (few GB) Do we need to copy the dataset for making plots?